zulia-server/src/main/java/io/zulia/server/index/ShardReader.java

```diff
@@ -508,7 +520,7 @@ else if (shardFacets == 0) {
 			SortFieldInfo sortFieldInfo = indexConfig.getSortFieldInfo(sortField);
 
 			if (sortFieldInfo == null) {
-				throw new IllegalArgumentException("Field <" + sortField + "> must be sortable");
+				throw new IllegalArgumentException("Field " + sortField + " must be sortable");
```

Comment:
Personally think it's good not to do the angle brackets anymore but I would use single quotes to still have a little distinction in the log.

Reply:
Yeah single quotes might be good.  Did not really want this change here but it got sucked in even though it was mostly done on another branch.